jaggeryapps/admin/acs.jag

```diff
@@ -114,17 +115,34 @@ if (samlResponse != null) {
     // This is an SLO request from the IDP.
     // Get the HTTP session object for the given session index, and invalidate it.
 
-    log.info("SLO request received.")
-    var index = sso.client.decodeSAMLLogoutRequest(sso.client.getSamlObject(samlRequest));
-
-    log.info("Session index of the SLO request is " + index);
-
-    var httpSessionForSessionIndex = application.get('httpSessions')[index];
-
-    if(httpSessionForSessionIndex){
-        log.info("Invalidating http session for the session index " + index);
-        httpSessionForSessionIndex.invalidate();
-        delete application.get('httpSessions')[index];
+    log.info("SLO request received.");
```

Comment:
Reason for this being in info level? better to add debug data to log and make it debug

Reply:
I will change it. Initially I didn't change it because it was already existing in the code.